overlord/servicestate/quota_handlers.go

```diff
@@ -749,6 +839,20 @@ func validateSnapForAddingToGroup(st *state.State, snaps []string, group string,
 	return nil
 }
 
+func quotaControlAffectedQuotas(t *state.Task) (quotas []string, err error) {
```

Comment:
nitpick: I'd rename all these functions to `affected{Quotas,Snaps}For<operation>`, that is `affectedQuotasForQuotaControl`, `affectedSnapsForQuotaControl`, `affectedQuotasForQuotaAddSnap`, etc.

Reply:
Fixed, thanks for the input!